Block/CheckoutComments.php

```diff
@@ -77,7 +77,7 @@ public function getOrderItemData()
         $orderItems = [];
 
         foreach ($order->getAllVisibleItems() as $item) {
-            $product = $item->getProduct();
+            $product = $this->config->getUseChildSku() ? $this->getChildProduct($item) : $item->getProduct();
```

Comment:
Refactor to use the same logic as here: https://github.com/turnto/magento2-turnto-socialcommerce/pull/10/files#diff-9be2c9c05649f07533f95396858e7fb9R477